bin/add_celltypes_to_sce.R

```diff
@@ -519,11 +527,19 @@ if (assign_consensus) {
 
     # get the full count
     reference_cell_count <- sum(sce$is_infercnv_reference)
+
+    # get hash for sorted concatenated barcodes
+    concat_barcodes <- paste(
+      sort(sce$barcodes[sce$is_infercnv_reference]),
+      collapse = ""
+    )
+    reference_cell_hash <- digest::digest(concat_barcodes, algo = "sha256")
   }
 }
 
 # export annotated object with cell type assignments
 readr::write_rds(sce, opt$output_sce_file, compress = "bz2")
 
-# export normal cell count
+# export normal cell count and hash
 readr::write_lines(reference_cell_count, opt$reference_cell_count_file)
+readr::write_lines(reference_cell_hash, opt$reference_cell_hash_file)
```

Comment:
Prevent trailing `\n` so we get empty files if empty. 
```suggestion
readr::write_file(reference_cell_count, opt$reference_cell_count_file)
readr::write_file(reference_cell_hash, opt$reference_cell_hash_file)
```